app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/itemlisting/VaultItemListingViewModel.kt

```diff
@@ -895,6 +899,14 @@ class VaultItemListingViewModel @Inject constructor(
                     ciphers = vaultData.cipherViewList,
                     matchUri = matchUri,
                 ),
+                fido2CredentialAutofillViewList = (vaultRepository
```

Comment:
💭 Could we put this filter logic in its own private function and use it here and on `line 930` too.

Reply:
I like that - thanks!